lightkurve/mast.py

```diff
@@ -38,7 +38,16 @@ def download_products(products):
     # Note: by default, MAST will only let us download "Minimum Recommended
     # Products" (MRPs), which do not include e.g. Target Pixel Files.
     # We need to set `mrp=False` to ensure MAST downloads whatever we want.
-    dl = Observations.download_products(products, mrp_only=False)
+
+    # check if download directory exists
+    if not os.path.isdir(os.path.expanduser('~')+'/.astropy'):
+        os.mkdir(os.path.expanduser('~')+'/.astropy')
```

Comment:
If you don't have write access this will break with a funny error message. Can you add an error message?